src/transformers/testing_utils.py

```diff
@@ -292,10 +292,14 @@ def require_intel_extension_for_pytorch(test_case):
     """
     Decorator marking a test that requires Intel Extension for PyTorch.
 
-    These tests are skipped when Intel Extension for PyTorch isn't installed.
+    These tests are skipped when Intel Extension for PyTorch isn't installed or it does not match current PyTorch
+    version.
```

Comment:
also note that it doesn't know what to do with pt-nightly and installs 1.12 version and then the tests get skipped as the version doesn't match.

it works fine with pt-1.12